src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -495,6 +495,7 @@ namespace AppInstaller::CLI::Workflow
             Workflow::GetDependenciesFromInstaller <<
             Workflow::ReportDependencies(Resource::String::InstallAndUpgradeCommandsReportDependencies) <<
             Workflow::ManagePackageDependencies(Resource::String::InstallAndUpgradeCommandsReportDependencies) <<
+            Workflow::EnableWindowsFeaturesDependencies <<

```

Comment:
This whole thing doesn't seem like the right way to handle dependencies.  At least, "download" is not consistent with "and also, handle getting all dependencies squared away".  So at least maybe change the name of this task?

Reply:
Moved this out of "download" to the step right before `InstallPackageInstaller`.

Reply:
Thanks for the explanation, I didn't know that COM allowed for multiple downloads. I moved the task back to the suggested location and added a TODO comment explaining the issue that needs to be addressed.